plugins/services/src/js/utils/QuotaUtil.ts

```diff
@@ -119,3 +119,15 @@ export function serviceTreeHasQuota(
     Object.keys(role.quota.limit).length > 0
   );
 }
+
+export function formatQuotaValueForDisplay(value: number): number {
+  //@ts-ignore
+  return +(Math.round(value + "e+2") + "e-2");
```

Comment:
This is so cool!

Reply:
I can't take credit I lifted it from the [Dashboard component](https://github.com/dcos/dcos-ui/blob/master/src/js/components/charts/ResourceTimeSeriesChart.js#L70-L72)

Reply:
interestingly it's not mentioned in [the spec](https://tc39.es/ecma262/#sec-math.round) but works across browsers. 🤔

would you object to adding a `parseFloat()` to get rid of the `ts-ignore`?

```
+(Math.round(parseFloat(value + "e+2")) + "e-2");
```

Reply:
We might want extracting the code into a Util or ui-kit component. So that our charts are always accurately rounded.

Reply:
`ui-kit` components could make sense since we do similar things in the other products.

Reply:
@pierrebeitz instead of `parseFloat` I just replaced the string concatenation with math.
`+(Math.round(value + "e+2") + "e-2")` -> `Math.round(value * 100) / 100` which is essentially the same thing right?

Reply:
I _think_ those are't the same... at least if we trust internet.

Reply:
There are subtle differences in that we might have rounding errors with the new solution. I *think* `+parseFloat(value).toFixed(2)` would be equivalent, more intention revealing and typesafe. :)

The `+` in front is for getting rid of trailing zeros.

Reply:
How many ways can you round a number in JS ;)

Reply:
Updated to `+value.toFixed(2)`